### PR TITLE
Show winrate interval in explore tab

### PR DESCRIPTION
**Pull request by Thaoden**
Decks in the Explore tab often show a high winrate while having been played only a few matches, which makes it difficult to properly assess their performance.

This PR uses the code that was already present for the My Decks tab to display the winrate interval next to the wins : losses ratio, together with the interval boundaries in the tooltip. The part where it says "more matches are needed" is left out, as it is usually not in the capabilities of the one using the Explore tab to play more matches with the deck (s)he's inspecting.

### CODE REVIEW

File: src/window_main/renderer-util.js [hunk 1]
```diff
@@ -760,6 +760,11 @@ function formatPercent(value, config = { maximumSignificantDigits: 2 }) {
   });
 }
 
+function formatWinrateInterval(lower, upper){
```
Comment (Lego6245) [inline src/window_main/renderer-util.js]: If you can make this `export function` and delete line 1078, that'd be great, as I'm meaning to get rid of the bulk export blocks at the end of files.

File: src/window_main/decks.js [hunk 2]
```diff
@@ -212,10 +213,7 @@ export function openDecksTab(_filters = {}, scrollTop = 0) {
       if (dwr.total >= 20) {
         // sample Size is large enough to use Wald Interval
         interval = formatPercent(dwr.interval);
-        tooltip = `${formatPercent(dwr.winrateLow)} to ${formatPercent(
-          dwr.winrateHigh
-        )} with 95% confidence
-(estimated actual winrate bounds, assuming a normal distribution)`;
```
Comment (Lego6245) [inline src/window_main/decks.js]: do we care about losing this text?

File: src/shared/util.js [hunk 3]
```diff
@@ -724,3 +724,8 @@ export function formatRank(rank) {
   }
   return `${rank.rank} ${rank.tier}`;
 }
+
+
+export function roundWinrate(x){
```
Comment (Lego6245) [inline src/shared/util.js]: I think this makes sense as a util, would like to be spot-checked.

**Comment by Lego6245** [comment]
Your build failed because of prettier, try running `npm run format` and you should get a green build.

File: src/window_main/explore.js [hunk 4]
```diff
@@ -613,13 +618,25 @@ function deckLoad(_deck, index) {
...
   );
+  if (_deck.mt >= 20) {
+    // sample Size is large enough to use Wald Interval
+    d.title = formatWinrateInterval(
+      roundWinrate(deckWinrate.winrate - deckWinrate.interval),
+      roundWinrate(deckWinrate.winrate + deckWinrate.interval)
+    );
+    d.innerHTML += `<i style="opacity:0.6;"> &plusmn; ${formatPercent(
+      roundWinrate(deckWinrate.interval)
+    )}</i>)`;
```
Comment (lusbenjamin) [inline src/window_main/explore.js]: may be an extra trailing `)` in the backtick here?

File: src/window_main/explore.js [hunk 5]
```diff
@@ -613,13 +618,25 @@ function deckLoad(_deck, index) {
...
+
+  let colClass = getWinrateClass(deckWinrate.winrate);
   d = createDiv(
     ["list_deck_record"],
-    `${_deck.mw}:${_deck.ml} <span class="${colClass}_bright">(${Math.round(
-      (100 / _deck.mt) * _deck.mw
-    )}%)</span>`
+    `${_deck.mw}:${_deck.ml} <span class="${colClass}_bright">(${formatPercent(
+      deckWinrate.winrate
+    )})</span>`
```
Comment (lusbenjamin) [inline src/window_main/explore.js]: my bad, the extra is prolly this one

File: src/window_main/decks.js (hunk 2 above)
Comment (lusbenjamin) [reply src/window_main/decks.js]: I thought the text just moved to the helper fn

Comment (Lego6245) [reply src/window_main/decks.js]: oh i see now, my brain must have skipped over it. ignore this, then :)

File: src/window_main/explore.js (hunk 5 above)
Comment (Lego6245) [reply src/window_main/explore.js]: If you're using vscode, colored bracket pairs extension is a godsend :)

File: src/window_main/renderer-util.js (hunk 1 above)
Comment (Thaoden) [reply src/window_main/renderer-util.js]: Will do.

File: src/window_main/explore.js (hunk 5 above)
Comment (Thaoden) [reply src/window_main/explore.js]: Still getting to grips with VS Code, the "whole" IDE VS has it all baked in...

Yeah, the `)` is a leftover from a copied `createDiv()`...

**Comment by Thaoden** [comment]
> Overall, LGTM! Thanks for contributing! I'm glad you updated the "my decks" tab too as part of this PR.

Well, yeah, no use in introducing a utility function and not using it ;)

Just pushed a new version.

File: src/window_main/explore.js (hunk 5 above)
Comment (Thaoden) [reply src/window_main/explore.js]: Alright, I'm completely lost now...

There was an extra `)` on line 638, that one is fixed.

The first bracket on line 628 closes the opening one from `formatPercent(`. The second one is plaintext (not interpreted) and closes the opening plaintext bracket just after `bright">`, all in all, it gives `( ${formatPercent(foo)} )`, so the bracket here is needed.

**Comment by Manuel-777** [comment]
Looks good!

**Comment by Thaoden** [comment]
Happy to see this merged :)

**Pull request merged**